core/src/main/kotlin/core/io/StandardMid.kt

```diff
@@ -36,8 +36,14 @@ object StandardMid {
             warnings,
         )
 
-        val tracks = midiTracks.drop(1).mapIndexed { index, midiTrack ->
+        val tracks = midiTracks.mapIndexed { index, midiTrack ->
             parseTrack(index, timeDivision, tickPrefix, midiTrack, params.defaultLyric)
+        }.toMutableList()
```

Comment:
I guess we can just filter all tracks to remove empty ones?
```kotlin
        val tracks = midiTracks.mapIndexed { index, midiTrack ->
            parseTrack(index, timeDivision, tickPrefix, midiTrack, params.defaultLyric)
        }.filter { it.notes.isNotEmpty() }
```

Reply:
I tried to keep old behaviors as much as possible. Is it okay to remove all empty tracks?

Reply:
Right, let's keep the original behavior 👍